### ISSUE TITLE
[QUESTION] 

### ISSUE THREAD

**Issue by gktval** [mono/SkiaSharp#975]
With .Net Core 3 coming up soon, has there been discussion on implementing .Net Core with SkiaSharp.Views? 

Right now I am using version 1.68.0 for my Net Core cross platform project. It gives me the warning "Package was restored using .NETFramework...". It still works, but seeing that .Net Core is supposed to replace any new and upcoming projects from being created in NetFramework, it would be important to have a functioning version of SkiaSharp.Views available.

**Comment by trovialdo** [comment]
Looks like this will be added in https://github.com/mono/SkiaSharp/pull/986

**Comment by mattleibow** [comment]
+1 on that title 😉 

This will also be 50% there with 1.68.1. I have separated the WPF and WinForms views into a separate NuGet package:
 - https://www.nuget.org/packages/skiasharp.views.wpf
 - https://www.nuget.org/packages/skiasharp.views.windowsforms

This will still have the warning, but should work much better without having to use bulky views package. Right now, it will pull in _both_ WPF and WinForms - and GTK.

If you are doing .NET Core on Linux and want views, we also got: https://www.nuget.org/packages/skiasharp.views.gtk3
